docs/config.md

```diff
@@ -255,6 +255,9 @@ Some ready-to-use trailer templates are available for frequently used trailers:
   using the committer info;
 * `format_gerrit_change_id_trailer(commit)` creates a "Change-Id" trailer
   suitable to be used with Gerrit. It is based Jujutsu's change id.
+  Note: If you simply set your trailers to this template,
+  you might get duplicate "Change-Id" trailers sometimes.
```

Comment:
I think it would help to explain why this occurs and what you should do in those cases. (Probably in the Gerrit FAQ entry, not here.)

The most common case that comes to mind for me is splitting a change. One change id will remain the same, and the other will change. In the second change, you'll get two Change-Ids without `contains_key` deduplication. Even with deduplication, the second change will end up with the same Change-Id as the first, which is not what you actually want. It would be nice to have some guidance about needing to decide which Change-Id you want to keep and which you want to regenerate.

I'm also happy to send a PR for this.